leetcode/src/5.c

```diff
@@ -0,0 +1,52 @@
+/**
+ * Find longest palindrome by traversing the string and checking how
+ * long palindrome can be constructed from each element by going left and right.
+ * Checking palindromes of types '..aa..' and '..bab..'
+ */
+
+#include <stdlib.h>
+#include <string.h>
```

Comment:
If we're keeping these, please document what they serve in this file (see the example below).

```c
#include <stdio.h>      /// for IO operations
#include <assert.h>    /// for assert
```

Reply:
This is done via new commit